db/migrations/20220607215900_add_resource_ownership_column.go

```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import (
+	logging "github.com/RedHatInsights/sources-api-go/logger"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// AddResouceOwnershipToApplicationTypes adds new column "resource_ownership" into table
+// "application_types".
+func AddResouceOwnershipToApplicationTypes() *gormigrate.Migration {
+	type ApplicationType struct {
+		ResourceOwnership string `gorm:"type:CHARACTER VARYING"`
```

Comment:
should not be here pointer as it is in `ApplicationType` in next commit ?

Reply:
sharp eye 😁 fixed